counsel.el

```diff
@@ -6182,6 +6182,12 @@ The buffers are those opened during a session of `counsel-switch-buffer'."
   "When non-nil, `counsel-switch-buffer' will preview virtual buffers."
   :type 'boolean)
 
+(defcustom counsel-switch-buffer-display-empty-buffer-when-no-preview nil
+  "When non-nil, `counsel-switch-buffer' will display an empty buffer when
+the current candidate is not previewed. When nil, `counsel-switch-buffer' 
+will instead display the buffer it was called from."
```

Comment:
Please make sure the first summary sentence of docstrings fits on a single line, as per Emacs conventions - [`(info "(elisp) Documentation Tips")`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html).

Please also end sentences with two spaces, as per the Emacs and Swiper `dir-locals-file` settings.